### ISSUE TITLE
xyz_loss increases during training

### ISSUE THREAD

**Issue by 3Der1999** [ShenhanQian/GaussianAvatars#37]
Hello, I'm running training code and find xyz_loss increases during training. Is this as expected?
![image](https://github.com/ShenhanQian/GaussianAvatars/assets/158052825/880046f0-877b-4c6e-9283-0c322b9f794c)
Could you please explain `losses['xyz'] = F.relu((gaussians._xyz*gaussians.face_scaling[gaussians.binding])[visibility_filter] - opt.threshold_xyz).norm(dim=1).mean() * opt.lambda_xyz` while `threshold_xyz = 1.`? I don't really understand it.
Thanks!

**Comment by ShenhanQian** [comment]
Hi, the increase of xyz_loss is expected as the Gaussians shift from the triangle to better capture the geometry of the subject.

You can find an explanation of this term by *Position loss with threshold* in Section 3.4 of our [paper](https://arxiv.org/html/2312.02069v2).
